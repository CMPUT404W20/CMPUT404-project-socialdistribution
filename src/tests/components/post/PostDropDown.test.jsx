import React from "react";
import { shallow } from "enzyme";
import PostDropdown from "../../../components/post/PostDropDown";

describe("Post DropDown Components", () => {
  it("should render correctly", () => {
    const component = shallow(<PostDropdown />);
    expect(component).toMatchSnapshot();
  });
});
