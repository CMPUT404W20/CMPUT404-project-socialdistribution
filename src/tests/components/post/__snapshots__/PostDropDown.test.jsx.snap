// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post DropDown Components should render correctly 1`] = `
<div>
  <DropdownItem
    as={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "displayName": "SafeAnchor",
        "render": [Function],
      }
    }
    disabled={false}
  >
    <img
      alt="pencil-icon"
      src="edit-pencil.svg"
    />
    <span
      className="post-dropdown-text"
    >
      Edit
    </span>
  </DropdownItem>
  <DropdownDivider
    role="separator"
  />
  <DropdownItem
    as={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "displayName": "SafeAnchor",
        "render": [Function],
      }
    }
    disabled={false}
  >
    <img
      alt="del-icon"
      src="del-bin.svg"
    />
    <span
      className="post-dropdown-text"
    >
      Delete
    </span>
  </DropdownItem>
  <DropdownDivider
    role="separator"
  />
  <DropdownItem
    as={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "displayName": "SafeAnchor",
        "render": [Function],
      }
    }
    disabled={false}
  >
    <img
      alt="link-icon"
      src="link.svg"
    />
    <span
      className="post-dropdown-text"
    >
      Copy link to post
    </span>
  </DropdownItem>
</div>
`;
